compile.c

```diff
@@ -12109,6 +12118,60 @@ ibf_load_object(const struct ibf_load *load, VALUE object_index)
     }
 }
 
+// Similar to ibf_load_object(), but specifically for CDHASH.
+static VALUE
+ibf_load_cdhash(const struct ibf_load *load, VALUE object_index)
+{
```

Comment:
Can we re-use `ibf_load_object()` instead of this code?
```C
  ibf_load_cdhash(){ hash = ibf_load_object(); RHASH_TBL_RAW(cdhash)->type = &cdhash_type; }
```

I didn't try this approach and it is tried by you before.